bpf/bpf_host.c

```diff
@@ -365,7 +365,7 @@ handle_ipv6_cont(struct __ctx_buff *ctx, __u32 secctx, const bool from_host,
 #ifdef ENABLE_IPSEC
```

Comment:
I added 
>A legitimate scenario where this might occur is during bootup of the local
node, before the IPsec secret is processed and plumbed into the datapath
via the ENCRYPT_MAP.

to the patch description. That's the best "real world" scenario I can think of where the local node wouldn't have a valid SPI. If that's the case, it's probably a very short window.

Reply:
If Cilium marks the node ready before `ENCRYPT_MAP` is plumbed, then that's a leak that can happen for a short time during any cluster scale up. It definitely sounds like it's worth an advisory to me; and I know several users who would want to be notified about it.